src/azul/service/manifest_service.py

```diff
@@ -510,7 +510,11 @@ def _get_entities(self, path: str, doc: JSON) -> List[JSON]:
         return entities
 
     def _drs_url(self, file):
-        return self.repository_plugin.drs_uri(file['drs_path'])
+        drs_path = file['drs_path']
+        if drs_path is None:
+            return None
```

Comment:
I don't think this is right. This would end up as a TSV cell containing `None`.

Reply:
or CSV cell

Reply:
It does not contain the text "None", if that's what you're worried about. It results in the empty string, i.e., not characters between the cell separators (`\t\t` in TSV). If that's not what you want here I don't know how to proceed.

Reply:
`\t\t` would be fine. `\tNone\t` would not. I'd like to see evidence for your claim, though.